lmdk/libraries/module_example/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+
+cmake_minimum_required(VERSION 3.20)
```

Comment:
I see nothing specific in this short file that requires 3.20, is there?

We already have a (too many?) `cmake_minimum_required()` elsewhere, I don't see why we need another one here.